.github/workflows/check.yml

```diff
@@ -32,9 +34,12 @@ jobs:
           docker system prune --force --all --volumes
 
       - uses: actions/checkout@v2
+      - name: Clean up
+        continue-on-error: true
```

Comment:
why would we continue on error? 

next is very simple and should never error

 cargo check --features=runtime-benchmarks --workspace --exclude integration-tests